### PR TITLE
Initial addition of Chronograf

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: deployments/chronograf/values.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  BASE_PATH: /chronograf
+  CUSTOM_AUTO_REFRESH: "1s=1000"
+  INFLUXDB_URL: "https://monitoring.lsst.codes"
+  INFLUXDB_ORG: "square"
+  # STATUS_FEED_URL:
+  # "https://lsst-sqre.github.io/argocd-efd/apps/chronograf/feeds/tucson-teststand.json" ??
+# INFLUX_TOKEN should be in the chronograf secret
+# This is separate from the oauth secret because we don't want all those
+# keys/values mounted into the deployment environment
+envFromSecret: chronograf
```
Comment (afausti) [inline deployments/chronograf/values.yaml:75]: Maybe reuse the `monitoring` secret which is already in Vault? so we put the InfluxDB token in one place.

File: deployments/chronograf/templates/vault-secret.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
+  name: chronograf
+  namespace: chronograf
+spec:
+  path: secret/k8s-operator/roundtable/chronograf
```
Comment (afausti) [inline deployments/chronograf/templates/vault-secret.yaml:7]: See comment below on reusing the `monitoring`  secret.

**Review by afausti — APPROVED** [review]
Looks good, in particular, the ingress and GH OAuth configuration seem right to me. I think this is optional in Roundtable but I would definitely add chronograf as a subchart of the monitoring app. I see several advantages in doing so. Everything would be deployed on the monitoring namespace and you'll see both InfluxDB and Chronograf resources under a single Argo CD application. You could put all secrets in the `monitoring` secret in Vault and create it just once in the monitoring namespace.

**Comment by athornton** [comment]
Oh!  Cool, yeah, it didn't occur to me that I could make it a subchart.  I'll rearrange that.  That makes a lot more sense.

**Pull request merged**